.circleci/config.yml

```diff
@@ -891,6 +891,8 @@ jobs:
             git status
             git clean -fd
             git status
+            git push https://${GITHUB_TOKEN}@github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git
```

Comment:
This only pushed the same branch onto itself, you're missing a merge

Reply:
This is intentional. I believe CircleCI is auto-merging the master branch